docs/src/main/paradox/kafka.md

```diff
@@ -120,5 +120,16 @@ Scala
 Java
 :  @@snip [KafkaDocExample.java](/examples/src/test/java/jdocs/kafka/KafkaDocExample.java) { #wordSource }
 
+## Mergeable Offset
 
+The offset type for a projection is determined by the @apidoc[SourceProvider] that's used.
+Akka Projections supports a variety of offset types.
+In most cases an event is associated with a single offset row in the projection implementation's offset store, but the @apidoc[KafkaSourceProvider$] uses a special type of offset called a @apidoc[MergeableOffset].
+
+@apidoc[MergeableOffset] allows us to read and write a map of offsets to the projection offset store.
+This is required because a subscription to consume from Kafka normally spans more than 1 Kafka Partition (see the [Apache Kafka documentation](https://kafka.apache.org/documentation/#intro_topics) for more information on Kafka's partitioning model).
+To begin consuming from Kafka using offsets stored in a projection's offset store we must provide the Kafka Consumer with a map of topic partitions to offset map (a @scala[`java.util.Map[org.apache.kafka.common, java.lang.Long]`]@java[`java.util.Map<org.apache.kafka.common, java.lang.Long>`]).
```

Comment:
Is this correct? What is `org.apache.kafka.common`?

They are also the same, using java API. In that case, we don't need to have @scala/@java tags.

Reply:
> Is this correct? What is `org.apache.kafka.common`?

```suggestion
To begin consuming from Kafka using offsets stored in a projection's offset store we must provide the Kafka Consumer with a map of topic partitions to offset map (a @scala[`java.util.Map[org.apache.kafka.common.TopicPartition, java.lang.Long]`]@java[`java.util.Map<org.apache.kafka.common.TopicPartition, java.lang.Long>`]).
```
I'm guessing it's `TopicPartition`.

> They are also the same, using java API. In that case, we don't need to have @scala/@java tags.

The brackets around the type differ: `[]` vs `<>` :-)

Reply:
I'll fix on a new PR

Reply:
https://github.com/akka/akka-projection/pull/365

Reply:
Good catch @renatocaval

Reply:
oh yeah, brackets!

Reply:
Ah, I guess I lost the `TopicPartition` type while trying to get the java/scala code toggle thing to work.  Thanks.